test/connect.spec.tsx

```diff
@@ -146,11 +201,17 @@ describe('connect', () => {
     });
 
     it('keeps @component behavior', () => {
-      instance = <ConnectedA myText='foo'/>;
+      instance = ConnectedA({myText: 'foo'});
       instance.append(<TextView/>);
       expect(instance.children().length).to.equal(0);
     });
 
+    it('does not allow connected component to be made a factory afterwards', () => {
+      const connected = connect(() => ({myText: ''}))(CustomComponentImpl);
```

Comment:
I think it'd be better to write like `state => ({myText: ''})` to make consistent with other `connect` methods that are used.

Reply:
That would look like a mistake since the parameter isn't needed.